.ci/.jenkins_ruby.yml

```diff
@@ -1,4 +1,4 @@
-RUBY_VERSION:
+VERSION:
```

Comment:
Being as generic as possible so the GH action in charge of generating the matrix can work